src/main/java/ch/naviqore/service/gtfs/nativecxx/raptor/NativeGtfsRaptor.java

```diff
@@ -0,0 +1,76 @@
+package ch.naviqore.service.gtfs.nativecxx.raptor;
+
+import lombok.extern.slf4j.Slf4j;
+
+import java.lang.foreign.*;
+import java.lang.invoke.MethodHandle;
+import java.nio.charset.StandardCharsets;
+
+@Slf4j
+public class NativeGtfsRaptor {
+
+    public static void main(String[] args) throws Throwable {
+        // not so nice.... Expecting an absolute path of the library: src/main/java/ch/naviqore/service/gtfs/nativecxx/raptor/library.dll
+        System.load(
```

Comment:
Exception in thread "main" java.lang.UnsatisfiedLinkError: Expecting an absolute path of the library: src\main\java\ch\naviqore\service\gtfs\nativecxx\raptor\library.dll

Reply:
Maybe put the DLL outside the java folder?

For example: `src/main/resources/native-libs/`